### ISSUE TITLE
Improve documentation

### ISSUE THREAD

**Issue by scarrazza** [qiboteam/qibosoq#50]
As discussed the documentation should include:
- more details about the interface
- a short example/demo without qibolab

**Comment by rodolfocarobene** [comment]
I think it's better to postpone this just after #51 and #53 are closed, since they will change a bit (maybe significantly) the object to send to the server and the internal workflow
